nowcasting_dataset/data_sources/pv/pv_data_source.py

```diff
@@ -368,7 +368,14 @@ def load_solar_pv_data(
     # Convert the pv_system_id column names from strings to ints:
     pv_power_df.columns = [np.int32(col) for col in pv_power_df.columns]
 
-    return pv_power_df.tz_localize("Europe/London").tz_convert("UTC").tz_convert(None)
+    if "passiv" in filename:
+        pv_power_df = pv_power_df.tz_localize("UTC").tz_convert(None)
```

Comment:
I might have misunderstood but does this line actually change anything :slightly_smiling_face: ?

Maybe it's sufficient to simplify this `if` block to something like:

```python
if "passive" not in filename:
    # Change PVOutput.org data to UTC and then convert to tz-naive.
    pv_power_df = pv_power_df.tz_localize("Europe/London").tz_convert("UTC").tz_convert(None)
```

Reply:
oh that is much better - good idea